src/app/locale/translations/no.json

```diff
@@ -0,0 +1,487 @@
+{
+    "optimized_checkout": {
+        "address": {
+            "address_line_1_label": "Adresse",
+            "address_line_1_required_error": "Adresse er påkrevd",
+            "address_line_2_label": "Leilighet/Bygg",
+            "address_line_2_required_error": "Leilighet/bygg er påkrevd",
+            "address_not_recognized_heading": "Vi gjenkjente ikke adressen din",
+            "consignment_address_updated_text": "Elementet ble tildelt den nye adressen.",
+            "city_label": "By",
+            "city_required_error": "By er påkrevd",
+            "company_name_label": "Firmanavn",
+            "company_name_required_error": "Firma er påkrevd",
+            "confirm_address_action": "Fortsett med gjeldende adresse",
+            "confirm_address_text": "Vi fant ingen treff for adressen du skrev inn. Vennligst bekreft adressen.",
+            "country_label": "Land",
+            "country_required_error": "Land er påkrevd",
+            "custom_required_error": "{label} er påkrevd",
+            "custom_min_error": "{label} må være større enn {min}",
+            "custom_max_error": "{label} må være mindre enn {max}",
+            "invalid_characters_error": "{label} inneholder ugyldige tegn",
+            "custom_valid_error": "{label} er ikke gyldig",
+            "edit_address_action": "Rediger adresse",
+            "enter_address_action": "Skriv inn en ny adresse",
+            "add_address_heading": "Legg til adresse",
+            "save_address_action": "Lagre adresse",
+            "first_name_label": "Fornavn",
+            "first_name_required_error": "Fornavn er påkrevd",
+            "last_name_label": "Etternavn",
+            "last_name_required_error": "Etternavn er påkrevd",
+            "phone_number_label": "Telefonnummer",
+            "phone_number_required_error": "Telefonnummer er påkrevd",
+            "postal_code_label": "Postnummer",
+            "postal_code_required_error": "Postnummer er påkrevd",
+            "save_in_addressbook": "Lagre denne adressen i adresseboken min.",
+            "select_country_action": "Velg et land",
+            "select_state_action": "Velg en delstat",
+            "state_label": "Delstat/provins",
+            "state_required_error": "Delstat/provins er påkrevd",
+            "select": "Velg",
+            "select_all": "Alle",
+            "select_none": "Ingen"
+        },
+        "billing": {
+            "billing_address_heading": "Fakturaadresse",
+            "billing_heading": "fakturering",
```

Comment:
seems weird that this is lowercase 🤷

Reply:
It's also being used in other languages. I'll change all of them in this PR. 
And leave a Jira comment for the translation team.

<img width="897" alt="image" src="https://user-images.githubusercontent.com/88361607/170617794-77d1fba5-02e2-435d-9aca-f7a8cd6accd0.png">